### PR TITLE
Fix two spelling mistakes in Spanish text.

### PR DESCRIPTION
**Pull request by jessgusclark**
Fixes two spelling errors in Spanish text. Please double-check the corrections as I am not a Spanish speaker.

### CODE REVIEW

File: src/common/translations/es.json [hunk 1]
```diff
@@ -94,7 +94,7 @@
       "recoveryPhrase": {
         "title": "Recovery Phrase",
         "note1": "Escriba o copie estas palabras",
-        "note2": "en el orden correcto y guárdelos",
+        "note2": "en el orden correcto y guárdalos",
```
Comment (jjtechuy) [inline src/common/translations/es.json:97]: The change should be from "guárdelos" to "guárdelas"

File: src/common/translations/es.json [hunk 2]
```diff
@@ -94,7 +94,7 @@
...
         "note3": "un lugar seguro"
```
Comment (farinalvaro) [inline src/common/translations/es.json:98]:
```suggestion
        "note3": "en un lugar seguro"
```

File: src/common/translations/es.json (hunk 1 above)
Comment (jessgusclark) [reply src/common/translations/es.json:97]: okay, I'll update it. My chrome suggested the one I added.

Comment (jessgusclark) [reply src/common/translations/es.json]: Thanks! fixes.

File: src/common/translations/es.json [hunk 3]
```diff
@@ -477,7 +477,7 @@
     },
     "guardPhrase": {
       "title": "Resguarde su frase de recuperación",
-      "body": "YSu frase de recuperación se compone de 12 palabras seleccionadas al azar. Escriba cuidadosamente cada palabra en el orden en que aparecen."
+      "body": "Su frase de recuperación se compone de 12 palabras seleccionadas al azar. Escriba cuidadosamente cada palabra en el orden en que aparecen."
```
[the remark this answers is not in the document]
Comment (jessgusclark) [reply src/common/translations/es.json]: thank you! Fixed.

File: src/common/translations/es.json (hunk 2 above)
Comment (jessgusclark) [reply src/common/translations/es.json]: Thanks fixed!

**Review by jjtechuy — APPROVED** [review]
LGTM

**Review by farinalvaro — APPROVED** [review]
LGTM

**Pull request merged**